server/datastore/mysql/software.go

```diff
@@ -917,7 +917,7 @@ func listSoftwareDB(
 				DetailsLink: fmt.Sprintf("https://nvd.nist.gov/vuln/detail/%s", cveID),
 				CreatedAt:   *result.CreatedAt,
 			}
-			if opts.IncludeCVEScores {
+			if opts.IncludeCVEScores && !opts.WithoutVulnerabilityDetails {
```

Comment:
Do we need !without here still?

Reply:
Unfortunately we do, because when we are filtering with a vulnerability `opts.IncludeCVEScores` will be `true`, and the json response will be large. However, if we put `!opts.WithoutVulnerabilityDetails`, it will add all the joins, but the json it has to serialize will still be the smaller version.

Reply:
Got it; thx for the clarification!